cypress/e2e/home_page.cy.ts

```diff
@@ -0,0 +1,19 @@
+describe("The Home Page", () => {
+  it("successfully loads", () => {
+    cy.visit("/");
+  });
+
+  it("contains a header", () => {
+    cy.visit("/");
+
+    cy.get("header").should("exist");
+  });
```

Comment:
This should be: 
```suggestion
  it("Should contain a header", () => {
    cy.visit("/");

    cy.get("header").should("exist");
  });
```

Same with the other one.